### PR TITLE
Update README for `ECS_INSTANCE_ATTRIBUTES`

### LINKED ISSUE #698: Fix README for ECS_INSTANCE_ATTRIBUTES

### PR DESCRIPTION
**Pull request by vsiddharth**
Fixes #698

Signed-off-by: Vinothkumar Siddharth <sidvin@amazon.com>

### Testing

- [x] Builds on Linux (`make release`)
- [x] Builds on Windows (`go build -out amazon-ecs-agent.exe ./agent`)
- [x] Unit tests on Linux (`make test`) pass
- [x] Unit tests on Windows (`go test -timeout=25s ./agent/...`) pass
- [x] Integration tests on Linux (`make run-integ-tests`) pass
- [ ] Integration tests on Windows (`.\scripts\run-integ-tests.ps1`) pass
- [ ] Functional tests on Linux (`make run-functional-tests`) pass
- [ ] Functional tests on Windows (`.\scripts\run-functional-tests.ps1`) pass

New tests cover the changes: 

### Licensing

This contribution is under the terms of the Apache 2.0 License:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ configure them as something other than the defaults.
 | `ECS_IMAGE_CLEANUP_INTERVAL` | 30m | The time interval between automated image cleanup cycles. If set to less than 10 minutes, the value is ignored. | 30m | 30m |
 | `ECS_IMAGE_MINIMUM_CLEANUP_AGE` | 30m | The minimum time interval between when an image is pulled and when it can be considered for automated image cleanup. | 1h | 1h |
 | `ECS_NUM_IMAGES_DELETE_PER_CYCLE` | 5 | The maximum number of images to delete in a single automated image cleanup cycle. If set to less than 1, the value is ignored. | 5 | 5 |
-| `ECS_INSTANCE_ATTRIBUTES` | `[{"stack": "prod"}]` | A list of custom attributes, in JSON form, to apply to your container instances. | `[]` | `[]` |
```
Comment (aaithal) [inline README.md]: Can you change default values as well? `[]` -> `{}`.

Comment (vsiddharth) [inline README.md]: Done 👍

File: README.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ configure them as something other than the defaults.
...
+| `ECS_INSTANCE_ATTRIBUTES` | `{"stack": "prod"}` | A list of custom attributes, in JSON form, to apply to your container instances. | `{}` | `{}` |
```
Comment (samuelkarp) [inline README.md]: Can you change the text a bit?  The word "list" implies that it's a JSON list, but it's really a map.

Comment (nmeyerhans) [inline README.md]: @vsiddharth Please address Sam's comments; otherwise this looks good.

Comment (vsiddharth) [inline README.md]: @nmeyerhans @samuelkarp Updated `list` to `map`.

**Pull request closed without merging**